lib/types.nix

```diff
@@ -27,6 +27,14 @@ let
           adding [$e] to the end of the key.
         '';
       };
+      persistent = lib.mkOption {
```

Comment:
Does this allow the user to set `persistent = true;` and `immutable = true;` at the same time? What is the desired behaviour of plasma-manager in that case?
(Imo this shouldn't be possible.)

Reply:
I agree, it shouldn't be possible. It's an easy fix luckily